src/scry/workspace.cr

```diff
@@ -9,16 +9,26 @@ module Scry
     property open_files
     property dependency_graph : Completion::DependencyGraph::Graph
 
+    @lookup_path : Array(String)
+
     def initialize(root_uri, process_id, max_number_of_problems)
       @root_uri = root_uri
       @process_id = process_id
       @max_number_of_problems = max_number_of_problems
       @open_files = {} of String => {TextDocument, Completion::MethodDB}
       @dependency_graph = Completion::DependencyGraph::Graph.new
+      @lookup_path = ENV["CRYSTAL_PATH"].split(":") + [@root_uri]
```

Comment:
Small note: Remember: If you're doing `+ [expr]` you're having 2 new arrays!

Reply:
What about `ENV["CRYSTAL_PATH"].split(":") << @root_uri` ?